x-pack/platform/plugins/private/upgrade_assistant/public/application/components/es_deprecations/deprecation_types/data_streams/flyout/steps/details/details_step.tsx

```diff
@@ -233,17 +234,31 @@ export const DataStreamDetailsFlyoutStep: React.FunctionComponent<{
               {!hasFetchFailed && !isCompleted && hasRequiredPrivileges && (
                 <EuiFlexItem grow={false}>
                   <EuiButton
-                    color={status === DataStreamReindexStatus.cancelled ? 'warning' : 'primary'}
-                    iconType={status === DataStreamReindexStatus.cancelled ? 'play' : undefined}
-                    onClick={startReindex}
+                    color={status === DataStreamMigrationStatus.cancelled ? 'warning' : 'accent'}
```

Comment:
I think this should just be an unfilled primary color button:

<img width="493" alt="Screenshot 2025-02-07 at 18 10 41" src="https://github.com/user-attachments/assets/5e8046c0-a8e3-4361-b020-650362b39d49" />